client/mumble-plugin/lib/io_UDPServer.cpp

```diff
@@ -758,12 +977,18 @@ void fgcom_shutdownUDPServer() {
 		return;
 	}
 
-	// send data
-	int len = sendto(sock, message.c_str(), strlen(message.c_str()), 0,
+	// send data with bounds checking
```

Comment:
`message` is (and will be) always a fixed string. No need to bounds check that - we have full authority over the bytes sent, at compile time.